### ISSUE TITLE
Updating to 0.13 causes `error:true` to be set on Express spans.

### ISSUE THREAD

**Issue by austinlparker** [lightstep/otel-launcher-node#33]
**Describe the bug**
After updating to 0.13.0, the `error` attribute is set to `true` on Express HTTP spans.

**To Reproduce**
Install express, launcher.

```
const {
  lightstep,
  opentelemetry,
} = require('lightstep-opentelemetry-launcher-node');

const sdk = lightstep.configureOpenTelemetry({
  accessToken: '<token>',
  serviceName: 'launcher-test',
});

sdk.start().then(() => {
  const express = require('express')
  const app = express()
  const port = 3000

  app.get('/', (req, res) => {
    res.send('Hello World!')
  })

  app.listen(port, () => {
    console.log(`Example app listening at http://localhost:${port}`)
  })
});
```

**Expected behavior**
A GET request should produce a span without an error.

**Additional context**

![image](https://user-images.githubusercontent.com/4140740/104532969-7392ca80-55df-11eb-9767-08cbc80b1132.png)

**Comment by mwear** [comment]
@obecny: Does v0.14.0 contain this PR: https://github.com/open-telemetry/opentelemetry-js/pull/1751? If not, will that fix this issue?

**Comment by obecny** [comment]
~@austinlparker pls use v0.14 of core and 0.12 of contrib, ignore v.0.13 of core~

hmm I mislead the version of launcher and version of otel. 
Anyway which version of express plugin are you using ?

**Comment by mwear** [comment]
Given that launcher includes @opentelemetry/plugins-node-core-and-contrib v0.12.0, I'd assume it's the version of the express plugin that it depends on (also v0.12.0).

**Comment by dobesv** [comment]
Having a similar problem with `koa` over here, although we are not using the `koa` plugin, just plain `http`.

**Comment by mwear** [comment]
This is fixed in the v0.13.1 release: https://www.npmjs.com/package/lightstep-opentelemetry-launcher-node/v/0.13.1.
